gaia-header.js

```diff
@@ -761,9 +755,11 @@ module.exports = component.register('gaia-header', {
   }
 
   [action=close] .action-button:before { content: 'close' }
-  [action=back] .action-button:before { content: 'back' }
   [action=menu] .action-button:before { content: 'menu' }
 
+  [action=back]:-moz-dir(ltr) .action-button:before { content: 'back' }
+  [action=back]:-moz-dir(rtl) .action-button:before { content: 'forward' }
```

Comment:
OK, the `rtl: true` trick works as expected. The good thing with this event is that it is a generic mutation observer, so we’re not depending on `l10n.js` here, but there are two trade-offs for me:
* performance-wise, a mutation observer is never free;
* shouldn’t we recalculate the title centering and font-size when the `localized` event is emitted anyway?

This first patch keeps the `rtl: true` logic and should fix most of the issues. If you don’t mind depending on the `localized` event, I can just listen to `localized` instead of `dirchanged` (and tweak the HTML example to fire a `localized` event when the direction is changed in the theme selector).